apps/service-portal/src/components/Layout/Layout.tsx

```diff
@@ -57,7 +57,7 @@ const Layout: FC = ({ children }) => {
         <Box as="main" component="main">
           <GridContainer className={styles.layoutContainer}>
             <GridRow>
-              <GridColumn span={'11/12'} className={styles.layoutGrid}>
+              <GridColumn span={'12/12'} className={styles.layoutGrid}>
```

Comment:
This should be 11/12 according to design. But we are discussing changes for the sidebar so the layout might change in the near future.

Reply:
Okey good point regarding desktop, but we still need to fix it for mobile to expand to the container else it looks the following in mobile. Notice the right margin.
<img width="509" alt="image" src="https://user-images.githubusercontent.com/112904566/196984699-b1525894-e2be-4c19-a3b8-f4fc353ac36e.png">

I will update the span to `{['12/12', '12/12', '12/12', '11/12']}`
Good with that? :)

Reply:
Yes thanks! Definitely, didn't spot the mobile part before 🙈